### PR TITLE
2016 survey post

### PR DESCRIPTION
**Pull request by aturon**
r? @erickt @rust-lang/core

### CODE REVIEW

**Comment by steveklabnik** [comment]
rendered https://github.com/aturon/blog.rust-lang.org/blob/survey-2016/_posts/2016-05-09-survey.md

File: _posts/2016-05-09-survey.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Completing this survey should take about 5 to 10 minutes, and is anonymous
+unless you choose to give us your contact information. We will be accepting
+submissions until June 8th, 2016.
+
+Please help us spread the word by sharing the above link on your social network
+feeds, at meetups, and around your office and other communities.
+
+Once the survey closes, we will summarize and visualize the results here on
+http://blog.rust-lang.org/. If you have any questions, please feel free to email
+the Rust Community team at community-team [at] rust-lang [dot] org.
```
Comment (steveklabnik) [inline _posts/2016-05-09-survey.md]: this should actually link with `mailto` imo

**Comment by aturon** [comment]
We seem to be missing a community team under the rust-lang org, so I'll ping directly:

cc @skade @steveklabnik @carols10cents @bstrie @erickt @brson @Manishearth

**Comment by erickt** [comment]
@aturon: we should include a thanks for all the people who helped develop and beta test the survey.

**Comment by aturon** [comment]
@erickt Sounds good! Do you want a generic thanks, or can you give me a list?

**Comment by aturon** [comment]
Added generic thanks, and a mailto link.

File: _posts/2016-05-09-survey.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+development priorities for the future.
+
+Completing this survey should take about 5 to 10 minutes, and is anonymous
+unless you choose to give us your contact information. We will be accepting
+submissions until June 8th, 2016. If you have any questions, please feel free to
+email the Rust Community team at
+[community-team@rust-lang.org](mailto:community-team@rust-lang.org).
+
+Please help us spread the word by sharing the above link on your social network
+feeds, at meetups, and around your office and other communities.
```
Comment (erickt) [inline _posts/2016-05-09-survey.md]: Remove the "and" in "and around your office"

**Comment by steveklabnik** [comment]
:shipit:

**Pull request merged**